### ISSUE TITLE
fix deprecations

### ISSUE THREAD

**Issue by stopfstedt** [ilios/frontend#2562]
these here:

```bash
$ ember s
DEPRECATION: Putting the "includePolyfill" option in "babel" is deprecated, please put it in "ember-cli-babel" instead.
DEPRECATION: Putting the "includePolyfill" option in "babel" is deprecated, please put it in "ember-cli-babel" instead.
```

**Comment by jrjohnson** [comment]
This will be fixed in #2557

**Comment by stopfstedt** [comment]
duplicate, then.
